centreon/src/Core/HostGroup/Infrastructure/Serializer/HostGroupResponseNormalizer.php

```diff
@@ -23,14 +23,18 @@
 
 namespace Core\HostGroup\Infrastructure\Serializer;
 
-use Core\HostGroup\Application\UseCase\FindHostGroups\Response\HostGroupResponse;
+use Core\HostGroup\Application\UseCase\FindHostGroups\HostGroupResponse;
+use Core\Infrastructure\Common\Api\HttpUrlTrait;
 use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
 use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
 
 final class HostGroupResponseNormalizer implements NormalizerInterface
 {
+    use HttpUrlTrait;
+    private const IMAGE_DIRECTORY = '/img/media/';
```

Comment:
This could be injected trough the configuration
https://github.com/centreon/centreon/blob/develop/centreon/config/services.yaml#L25

Example of configuration with injected configuration variable:
https://github.com/centreon/centreon/blob/develop/centreon/config/packages/Centreon.yaml#L554-L556